src/traits.rs

```diff
@@ -0,0 +1,22 @@
+
+/// Convenient methods for webdriver capabilities
+pub trait CapabilitiesExt {
```

Comment:
I think maybe we should make this change in a separate PR, just so that they are easier to review. What do you think?

Reply:
Ok, we can do that. That should make it easier for others to see the changes, I'll remove it from this.